client/src/text/pl.json

```diff
@@ -0,0 +1,15 @@
+{
+  "allowed_filetypes": "Allowed filetypes!",
```

Comment:
Polish is very excited! there are a lot of exclamations! but otherwise it seems mostly like English! :)